starknet/block.go

```diff
@@ -45,13 +46,29 @@ func (b *Block) GasPriceETH() *felt.Felt {
 	return b.GasPriceLegacy
 }
 
-func (b *Block) GasPriceSTRK() *felt.Felt {
+func (b *Block) L1GasPriceSTRK() *felt.Felt {
 	if b.L1GasPrice != nil {
 		return b.L1GasPrice.PriceInFri
 	}
 	return b.GasPriceFRI
 }
 
+// TODO: Fix when we have l2 gas price
```

Comment:
what did you mean by that?

Reply:
Now the feeder gateway doesn't send this field. It can be removed once we have it from the FGW and have field validation in the proto fields.